easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2106-foss-2021a.eb

```diff
@@ -0,0 +1,45 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'OpenFOAM'
+version = 'v2106'
+
+homepage = 'https://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://sourceforge.net/projects/openfoam/files/%(version)s/']
+sources = [SOURCE_TGZ]
+patches = [
+    ('OpenFOAM-v2012-cleanup.patch', 1),
+    'OpenFOAM-v1906-wmake-ompi.patch',
+]
+checksums = [
+    'e765881451d8d15ce33caa28850cdcc4534b8c4590e08ee6011bcafd11216940',  # OpenFOAM-v2106.tgz
```

Comment:
I'm getting a different SHA256 checksum for `OpenFOAM-v2106.tgz`: `11e41e5b9a253ef592a8f6b79f6aded623b28308192d02cec1327078523b5a37` 

Was there a sneaky re-release of OpenFOAM v2106 without a proper version bump? (@olesenm?!)

@robert-mijakovic Do you still have the original source tarball you were using, with checksum `e765881451d8d15ce33caa28850cdcc4534b8c4590e08ee6011bcafd11216940`?

Reply:
The SHA256 in the Spack packages agrees with what I'm seeing, so I'll change this...

https://github.com/spack/spack/blob/e911f8ab3cfc073f77ea874871a95cb63dd928f4/var/spack/repos/builtin/packages/openfoam/package.py#L268